Headers/DebugServer2/Target/Darwin/MachOProcess.h

```diff
@@ -31,7 +31,7 @@ class MachOProcess : public ds2::Target::POSIX::Process {
 public:
   virtual ErrorCode getSharedLibraryInfoAddress(Address &address);
   virtual ErrorCode enumerateSharedLibraries(
-      std::function<void(SharedLibraryInfo const &)> const &cb);
```

Comment:
If you add `override`, you should remove the `virtual` keyword here. You can skip this and we can do it in a different commit if you want.